### DISCUSSION TITLE
No access to the articles anymore/Vantage Child? Gutemberg? CSS?

### DISCUSSION THREAD

**Discussion by ChristinePlume** [WordPress/gutenberg#40196] [Q&A]
I am trying to help a friend with his website based on WordPress.
For 1 month approximately, he can neither
- publish a new article
- go back to the previous articles and modify them.
When you click on the menu to watch any article, nothing appears but a white screen with an error (see my screenshot).
He uses the block editor. I modified in order that the editor was the classic editor: then we can click on the articles but then we get only the title and no content anymore.
What's the problem? What do we have to do?
He uses Gutemberg. I disabled it but it doesn't modify anything.
I tried another theme than Vantage Child but the problem remains.
I checked the website health and... there is no problem found...
![ActuSS](https://user-images.githubusercontent.com/103139468/162493997-7318afd8-64af-4509-9170-9e2b7e821a98.jpg)

...

Many thanks for your help...

**Comment by talldan** [comment]
@ChristinePlume Please could you copy the error and paste it here.

Have you tried deactivating other plugins?

**Reply by ChristinePlume** [reply]
Yes I did. It seems now, after having checked everything for 15 days and tried many things, including reinstalling WordPress, that the hoster itself (Nuxit) made a mistake. If it's useful, I can post the informations concerning the way we will have solved the problem, when it's done.
